config_parser.go

```diff
@@ -22,6 +27,23 @@ func (r realConfigParser) parseConfig() config {
 	return cfg
 }
 
+func (r realConfigParser) expandTestsPathPattern(cfg config) []string {
+	var result []string
+	if cfg.TestsPathPattern == "" {
+		return result
+	}
+	rootPath := cfg.ProjectLocation + string(os.PathSeparator)
+	glob, err := doublestar.Glob(rootPath + cfg.TestsPathPattern)
```

Comment:
why not filepath.Join()? manually constructing a file path is dangerous.

Reply:
Didn't know such thing did exist 😓